### PR TITLE
SGE Addersgall / WHM Lily options

### PR DESCRIPTION
**Pull request by vond82**
Since this has been requested:
- Added Partial Fill Color options for SGE Addersgall & WHM Lily bars
- Separated Addersting config from Addersgall a bit
- Added glow options for when Blood Lily bar is full (Misery available)

### CODE REVIEW

File: DelvUI/Interface/Jobs/SageHud.cs [hunk 1]
```diff
@@ -179,7 +179,7 @@ public new static SageConfig DefaultConfig()
         );
 
         [NestedConfig("Addersting Bar", 40)]
-        public AddersgallBarConfig AdderstingBar = new AddersgallBarConfig(
+        public AdderstingBarConfig AdderstingBar = new AdderstingBarConfig(
```
Comment (Tischel) [inline DelvUI/Interface/Jobs/SageHud.cs:182]: Doesn't this break when updating?

File: DelvUI/Interface/Jobs/WhiteMageHud.cs [hunk 2]
```diff
@@ -215,14 +216,14 @@ public new static WhiteMageConfig DefaultConfig()
...
-        public ChunkedBarConfig LilyBar = new ChunkedBarConfig(
+        public LilyBarConfig LilyBar = new LilyBarConfig(
             new(-64, -32),
             new(126, 20),
             new PluginConfigColor(new(0f / 255f, 64f / 255f, 255f / 255f, 100f / 100f))
         );
 
         [NestedConfig("Blood Lily Bar", 35)]
-        public ChunkedBarConfig BloodLilyBar = new ChunkedBarConfig(
+        public BloodLilyBarConfig BloodLilyBar = new BloodLilyBarConfig(
```
Comment (Tischel) [inline DelvUI/Interface/Jobs/WhiteMageHud.cs:226]: Or these?

**Review by Tischel — APPROVED** [review]

**Comment by Tischel** [comment]
Makes sure you test what happens when updating or when importing a profile with these new types on the bars.
Might break, not sure tho.

File: DelvUI/Interface/Jobs/WhiteMageHud.cs (hunk 2 above)
Comment (vond82) [reply DelvUI/Interface/Jobs/WhiteMageHud.cs:226]: I was expecting I'd need to add converter stuff, but everything worked great as is even going back and forth between the builds

**Pull request merged**